status-im/status-react
# feat(swap): setup swap ui

fixes #20331 

### Summary

This PR implements Setup Swap screen, no backend integration yet and some data is hardcoded for now, but UI should be functional and most of the logic should be in place

<img src="https://github.com/user-attachments/assets/8a9f058e-c324-49bf-a7de-67b1f48e9983" width="300" />

#### Platforms

- Android
- iOS

#### Areas that maybe impacted

##### Functional

- wallet / transactions

### Steps to test

- Open Status
- Log in
- Open wallet
- Select an account
- Tap on Swap
- Select an asset in asset to pay screen
- Select Setup swap button
- Verify UI looks as expected

status: ready
